js/apps/admin-ui/src/ForbiddenSection.tsx

```diff
@@ -11,11 +11,17 @@ export const ForbiddenSection = ({
   permissionNeeded,
 }: ForbiddenSectionProps) => {
   const { t } = useTranslation();
-  const count = Array.isArray(permissionNeeded) ? permissionNeeded.length : 1;
+  const permissionNeededArray = Array.isArray(permissionNeeded)
+    ? permissionNeeded
+    : [permissionNeeded];
+  const count = permissionNeededArray.length;
 
   return (
     <PageSection>
-      {t("forbidden", { count })} {permissionNeeded}
+      {t("forbidden", { count })}{" "}
+      {permissionNeededArray
+        .filter((p) => typeof p === "string")
+        .map((p) => String(p))}
```

Comment:
This would work but the compiler complains due to _A type predicate's type must be assignable to its parameter's type. Type 'string' is not assignable to type 'AccessType'._ error since `AccessType` now contains the `AccessTypeFunc`. Is there any other way to shorten it? I'm happy to shorten or keep as is.

Reply:
Just tested this, and it looks like we can get away with calling `toString()`:

```ts
{permissionNeededArray.map((p) => p.toString())}
```

We don't need to filter this array right? According to the type information there should be no `undefined` values.

Reply:
That's a good idea. The filter was there to filter out the functions only but it works with `toString` nicely. I'll change that accordingly. Thanks for the tip.